src/pages/AppConfigure.vue

```diff
@@ -13,8 +15,17 @@ function switchTab(tab: ConfigureTab) {
 
 <template>
   <div class="container">
-    <main>
+    <VStack gap="2rem">
       <ConfigureSidebar :current-tab="currentTab" @switch-tab="switchTab" />
-    </main>
+      <main>
+        <GeneralSettings v-if="currentTab === 'general'" />
```

Comment:
We want to create sub-routes for each section of the sidebar. We can add child routes in the /config route and use a router view here.